var/spack/repos/builtin/packages/atmi/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Atmi(CMakePackage):
+    """Asynchronous Task and Memory Interface, or ATMI, is a runtime framework
+       and programming model for heterogeneous CPU-GPU systems. It provides a
+       consistent, declarative API to create task graphs on CPUs and GPUs
+       (integrated and discrete)."""
+
+    homepage = "https://github.com/RadeonOpenCompute/atmi"
+    url      = "https://github.com/RadeonOpenCompute/atmi/archive/rocm-3.5.0.tar.gz"
+
+    maintainers = ['srekolam', 'arjun-raj-kuppala']
+
+    version('3.5.0', sha256='3fb57d2e583fab82bd0582d0c2bccff059ca91122c18ac49a7770a8bb041a37b')
+
+    variant('build_type', default='Release', values=("Release", "Debug"), description='CMake build type')
+    depends_on('cmake@3.5.2', type='build')
```

Comment:
```suggestion
    depends_on('cmake@2.8:', type='build')
```
There's no reason to force a particular version of CMake is there? The `CMakeLists.txt` says 2.8+ is the only requirement.

Reply:
@adamjstewart , we will look into cmakeLists.txt for each recipe and modify the above . Thanks for your inputs.

Reply:
@adamjstewart  ,today I tried to change as per the cmakelist.txt for each recipe but it is creating some conflicts 
Like hsakmt-roct  and hsa-rocr-dev require cmake-3.5  ; llvm-amdgpu requires cmake 3.4.3 and comgr - cmake 3.2.0;
rocm-device-libs -cmake 3.4.3 and atmi - cmake 2.8
Say for atmi, it is dependent on hsakmt-roct and hsa-rocr-dev and they require cmake 3.5 ..whereas cmakelists.txt for atmi mentions it requires cmake 2.8 .. this seems to create conflict when we try to concretize the spec. 
I am thinking i will force a particular version of cmake unless there is any solution. let me know what you think.

Reply:
The solution is to add a colon at the end: `cmake@2.8:`. This means CMake 2.8+. If all of your packages use the ending colon, then Spack will use any version that satisfies all of them, in this case likely the newest version.